### PR TITLE
Add positioned badges example to docs

### LINKED ISSUE #34131: Suggestion: add a corner label layout example on the badge page
https://getbootstrap.com/docs/5.0/utilities/position/#examples
https://getbootstrap.com/docs/5.0/components/badge/

Add a corner label example to the badge page to use the new positioning class for layout.

### PR DESCRIPTION
**Pull request by jjoel1630**
Update the documentation for the badges in the Bootstrap 5.0 website. Adds a new way to show notifications on elements using badges. The attached image shows what it looks like. 

image

---

Closes #34131

Preview: https://deploy-preview-34143--twbs-bootstrap.netlify.app/docs/5.0/components/badge/

### CODE REVIEW

File: site/content/docs/5.0/components/badge.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ toc: true
 Badges scale to match the size of the immediate parent element by using relative font sizing and `em` units. As of v5, badges no longer have focus or hover styles for links.
 
 {{< example >}}
+
```
Comment (ffoodd) [inline site/content/docs/5.0/components/badge.md:14]: Useless change

File: site/content/docs/5.0/components/badge.md [hunk 2]
```diff
@@ -23,7 +24,7 @@ Badges can be used as part of links or buttons to provide a counter.
 
 {{< example >}}
 <button type="button" class="btn btn-primary">
-  Notifications <span class="badge bg-secondary">4</span>
+Notifications <span class="badge bg-secondary">4</span>
```
Comment (ffoodd) [inline site/content/docs/5.0/components/badge.md:27]: Useless change

File: site/content/docs/5.0/components/badge.md [hunk 3]
```diff
@@ -33,8 +34,27 @@ Unless the context is clear (as with the "Notifications" example, where it is un
 
 {{< example >}}
 <button type="button" class="btn btn-primary">
-  Profile <span class="badge bg-secondary">9</span>
-  <span class="visually-hidden">unread messages</span>
+Profile <span class="badge bg-secondary">9</span>
+<span class="visually-hidden">unread messages</span>
+</button>
+{{< /example >}}
```
Comment (ffoodd) [inline site/content/docs/5.0/components/badge.md:40]: Useless change

File: site/content/docs/5.0/components/badge.md [hunk 4]
```diff
@@ -33,8 +34,27 @@ Unless the context is clear (as with the "Notifications" example, where it is un
...
+Additionally, we can style the badge to show a notification in a much nicer way. This method is traditionally seen in mobile apps.
+
+{{< example >}}
+<button type="button" class="btn btn-primary position-relative">
+Alerts
+<span class="position-absolute badge start-100 rounded-circle border border-light translate-middle top-0 bg-danger pb-2">
+<span style="font-size: 0.8em;">10</span>
+</span>
+</button>
+{{< /example >}}
```
Comment (ffoodd) [inline site/content/docs/5.0/components/badge.md:51]: I'd be in favor of adding a single example (the second): this one looks a bit off with border and not really rounded. Moreover having inline style to set font-size seems like a bad idea to me, and at least should not be suggested in our examples.

image

The second example works great and is really good :)

**Review by ffoodd — CHANGES_REQUESTED** [review]
You have a few unrelated changes in there, and I'd recommend to only keep the second example you added: it works perfectly and elegantly, whereas the first one seems off.

File: site/content/docs/5.0/components/badge.md (hunk 4 above)
Comment (jjoel1630) [reply site/content/docs/5.0/components/badge.md:51]: Hello @ffoodd. Thank you for your suggestions, I will fix them as soon as I can.

**Comment by jjoel1630** [comment]
Hello @ffoodd, I have added the changes that you suggested. Please let me know if it looks fine to you.

**Comment by mdo** [comment]
@ffoodd I rewrote some of this to clear it up. Let me know what you think.

**Comment by ffoodd** [comment]
Looks great and is indeed clearer, thanks!

**Review by ffoodd — APPROVED** [review]

**Review by mdo — APPROVED** [review]

**Pull request merged**